.github/workflows/CI.yml

```diff
@@ -17,7 +17,7 @@ on:
     branches:
       - main
       - release/v*
-      - dev
+      - dev/*
```

Comment:
We still have a `dev` branch even if it isn't really in use. Should we still have the CI run on it? This change will remove that.

Reply:
There are two ways for us to change it: one is to rename the branch as dev/ and second is that change the prefix pattern to dev*.

Reply:
The other way is to just have two entries
```
- dev
- dev/*
```

Reply:
Reasonable. Fixed it.